js/markdown-stream/markdown-stream.scss

```diff
@@ -0,0 +1,51 @@
+@use "highlight_styles" as highlight_styles;
+
+/* Code highlighting (for both light and dark mode) */
+@include highlight_styles.atom_one_light;
+[data-bs-theme="dark"] {
+  @include highlight_styles.atom_one_dark;
+}
```

Comment:
Not something to handle in this PR, but we should anticipate highlighting styles being something that people will want to configure.